src/middlewared/middlewared/plugins/pool.py

```diff
@@ -3141,8 +3141,8 @@ def unlock(self, job, id, options):
             else:
                 unlocked.append(name)
 
-        if options['restart_related_attachments'] and not failed:
-            j = self.middleware.call_sync('pool.dataset.restart_related_attachments', self.__attachments_path(dataset))
+        if options['start_related_attachments'] and not failed:
```

Comment:
Not sure it any clearer with "start" ?

We will likely be restarting services because we may have shares that are enable for other pools/datasets?